Cargo.lock

```diff
@@ -121,9 +121,9 @@ checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
 [[package]]
 name = "bitflags"
-version = "2.2.1"
+version = "2.3.1"
```

Comment:
Bitflags 2.3.0 deprecates the `BitFlags` trait in favor of `Flags`: https://github.com/bitflags/bitflags/releases/tag/2.3.0.

It's likely we can still update, but it may be better to pull this out and assign someone to do the "migration".

Reply:
reverted in https://github.com/gravitational/teleport/pull/26660/commits/d619ec476dbfae8d8f7dfc8de32e6ca5418e08cf

Reply:
Many thanks!